src/components/SignerPanel/SigningStatus.js

```diff
@@ -141,9 +144,7 @@ class SigningStatus extends React.Component {
 }
 
 const Status = styled.div`
-  background: ${({ background }) => background};
-  height: 360px;
-  margin-bottom: ${3 * GU}px;
+  height: 300px;
```

Comment:
Let's also `GU`ify this! 😃 (I wonder if `38 * GU` might work?)
```suggestion
  height: ${37.5 * GU}px;
```

Reply:
Thanks for these man, I've provided whole values to stick to increments defined by the grid 👍

Reply:
Gotcha, no problem! It's looking 🔥